AutomatedTesting/Gem/PythonTests/terrain/EditorScripts/TerrainPhysicsCollider_ChangesSizeWithAxisAlignedBoxShapeChanges.py

```diff
@@ -0,0 +1,85 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+#fmt: off
+class Tests():
+    create_test_entity                      = ("Entity created successfully",    "Failed to create Entity")
+    add_axis_aligned_box_shape              = ("Axis Aligned Box Shape component added", "Failed to add Axis Aligned Box Shape component")
+    add_terrain_collider                    = ("Terrain Physics Heightfield Collider component added", "Failed to add a Terrain Physics Heightfield Collider component")
+    box_dimensions_changed                  = ("Aabb dimensions changed successfully", "Failed change Aabb dimensions")
+    configuration_changed                   = ("Terrain size changed successfully", "Failed terrain size change")
+    no_errors_and_warnings_found            = ("No errors and warnings found",   "Found errors and warnings")
+#fmt: on
+
+def TerrainPhysicsCollider_ChangesSizeWithAxisAlignedBoxShapeChanges():
+    """
+    Summary:
+    Test aspects of the TerrainHeightGradientList through the BehaviorContext and the Property Tree.
+
+    Test Steps:
+    Expected Behavior:
+    The Editor is stable there are no warnings or errors.
+
+    Test Steps:
+     1) Load the base level
+     2) Create test entity
+     3) Start the Tracer to catch any errors and warnings
+     4) Add the Axis Aligned Box Shape and Terrain Physics Heightfield Collider components
+     5) Change the Axis Aligned Box Shape dimensions
+     6) Chaeck the Heightfield provider is returning the correct size
+     7) Verify there are no errors and warnings in the logs
+
+
+    :return: None
+    """
+
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.utils import TestHelper as helper
+    from editor_python_test_tools.utils import Report
+    from editor_python_test_tools.utils import Tracer
```

Comment:
nit: combine imports into one to avoid cluttering:
```suggestion
    from editor_python_test_tools.utils import TestHelper as helper
    from editor_python_test_tools.utils import Report, Tracer
```

Reply:
Done